papers/ring_vrf/pedersen_vrf.tex

```diff
@@ -1,18 +1,18 @@
-\section{Ring VRFs from the Pedersen VRF}
+\section{Our Ring VRF Construction}
 \label{sec:pederson_vrf}
 
-An EC VRF like \cite{nsec5,VXEd25519,draft-irtf-cfrg-vrf-10} consists of
+Our ring VRF construction is based on EC VRF  \cite{nsec5,VXEd25519,draft-irtf-cfrg-vrf-10}. EC VRF consists of
```

Comment:
It's true my opening sentence is weak, but it[s not an opening sentence of a intro.  I donno..

Reply:
I just added a **tiny** sentence that explains why we talk about ECvRF now. You should not think or worry about this much with these little additions since the paper requires more important things to worry about.

Reply:
I've done roughly this.